src/src/Commands/InternalCommands.cpp

```diff
@@ -293,6 +300,7 @@ bool executeInternalCommand(const char *cmd, struct EventStruct *event, const ch
     #endif // ifndef BUILD_NO_DIAGNOSTIC_COMMANDS
         COMMAND_CASE_R(   "settings", Command_Settings_Print, 0); // Settings.h
       }
+      COMMAND_CASE_A("status", Command_GPIO_Status,          2); // GPIO.h
```

Comment:
Maybe rename this command to "GPIOstatus" as "status" is a bit too generic?

Reply:
ok for me, but it will be a breaking change, as this command was already in the original plugin.

Reply:
Ah ok, then leave it.